csharp/private/rules/library.bzl

```diff
@@ -91,6 +92,9 @@ csharp_library = rule(
             doc = "The standard set of assemblies to reference.",
             default = "@net//:StandardReferences",
         ),
+        "internals_visible_to": attr.string_list(
```

Comment:
It would be cooler if this were `attr_list`. I can't remember if its OK for the target graph to have cycles though.... the action graph certainly can't but that wouldn't be an issue.

Reply:
Doesn't look like this is possible, so strings it is.